src/controllers/requestController.js

```diff
@@ -213,8 +215,9 @@ class RequestController {
         request = await RequestService.retrieveManagerRequest(directReportIds, id);
       }
 
-      // eslint-disable-next-line max-len
-      if (isRequester) request = await RequestService.retrieveOneRequest({ user_id: userData.id, id });
+      if (isRequester) {
+        request = await RequestService.retrieveOneRequest({ user_id: userData.id, id });
```

Comment:
And this too, why can't we refactor this code into a function and specify the values we need from the requester

Reply:
We need the user_id to find a request that was created by that user and the ID becomes the id of the request so those are the values needed